include/z64math.h

```diff
@@ -76,6 +76,9 @@ typedef struct {
     /* 0x06 */ s16 yaw;    // azimuthal angle
 } VecSph; // size = 0x08
 
+#define BINANG_ROT180(angle) ((s16)((angle) - 0x7FFF))
+#define BINANG_SUB(a, b) ((s16)((a) - (b)))
```

Comment:
Just a side note but I would agree with removing these macros. I think the codebase is way too inconsistent/hardcoded to try and abstract the fact that angles use s16 types and value ranges.

Reply:
+1 from me, particularly for BINANG_SUB. 
I think the rotate 180 would not be immediately obvious to someone who isnt used to the pattern. So it provides a benefit in that sense. But I dont like the idea of macro-ifying the operation in general